encrypted-media-respec.html

```diff
@@ -699,8 +747,8 @@
                 <ol>
                   <li><p>Let <var>session type</var> be the value.</p></li>
                   <li>
-                    <p>If the <var>implementation</var> does not support <var>session type</var> in combination with <var>accumulated configuration</var>, return <code>null</code>.</p>
-                    <p>Unsupported combinations include but are not limited to a <var>session type</var> for which the <a def-id="is-persistent-session-type-algorithm"></a> algorithm returns <code>true</code> when <var>accumulated configuration</var>'s <a def-id="option-persistentState"></a> value is <a def-id="requirement-not-allowed"></a>.</p>
+                    <p>If the <var>implementation</var> does not support <var>session type</var> in combination with <var>accumulated configuration</var> and <var>restrictions</var>, return <code>NotSupported</code>.</p>
+                    <p>Unsupported combinations include but are not limited to a <var>session type</var> for which the <a def-id="is-persistent-session-type-algorithm"></a> algorithm returns <code>true</code> when <var>accumulated configuration</var>'s <a def-id="option-persistentState"></a> value is <a def-id="requirement-not-allowed"></a> or <var>accumulated configuration</var>'s <a def-id="option-persistentState"></a> value is <a def-id="requirement-optional"></a> but persistent state is disallowed by <var>restrictions</var>.</p>
```

Comment:
Should we just address this (and DI) in the "optional" cases above? For example, replace "Continue" with "If _restrictions_ prevents..., set the value to `"not-allowed"`." That seems more explicit and maintainable.